src/osvr/Util/LogRegistry.cpp

```diff
@@ -135,7 +152,8 @@ namespace util {
             spdlog::set_pattern(DEFAULT_PATTERN);
```

Comment:
I think `set_pattern()` only throws in OOM cases. `set_level()` is just a bunch of assignments of the new level to replace the old level.